content/en/docs/tasks/security/authn-policy/index.md

```diff
@@ -19,6 +19,13 @@ the underlying concepts in the [authentication overview](/docs/concepts/security
 [installation steps](/docs/setup/install/kubernetes/#installation-steps), or set `global.mtls.enabled` to false using
 [Helm](/docs/setup/install/helm/)).
 
+{{< warning >}}
+This tutorial assume you're using Istio 1.4 with auto mutual TLS turned on via Helm flag
```

Comment:
This is the documentation for 1.4 so we don't need to mention 1.4 or 1.3 in the docs. Just assume the user is on 1.4+ and write the text accordingly.